ffi/src/lib.rs

```diff
@@ -1033,6 +1040,7 @@ pub unsafe extern "C" fn command(
     args_len: *const c_ulong,
     route_bytes: *const u8,
     route_bytes_len: usize,
+    span_ptr: u64,
```

Comment:
Can you change the type?
```suggestion
    span_ptr: *const Span,
```